sql/core/src/test/resources/tpcds/q32.sql

```diff
@@ -1,4 +1,4 @@
-SELECT 1 AS `excess discount amount `
+SELECT sum(cs_ext_discount_amt) AS `excess discount amount `
```

Comment:
The change looks good, but do we need the empty space after `amount`?

Reply:
Removed the extra space